INSTALL.md

```diff
@@ -14,14 +14,14 @@ Following are the steps to install the above dependencies for few major distribu
 ### For CentOS, RHEL, Fedora
 
 ```
-sudo yum install python python-pip build-essentials python-devel
+sudo yum install python python-pip build-essential python-devel
```

Comment:
@nehaljwani I have edited the package name here too. Is this what you were mentioning earlier ? 
Kindly clarify the same.